static/app/components/profiling/aggregateFlamegraphPanel.tsx

```diff
@@ -14,31 +18,66 @@ export function AggregateFlamegraphPanel({transaction}: {transaction: string}) {
 
   const isEmpty = data?.shared.frames.length === 0;
   return (
-    <ProfileGroupProvider type="flamegraph" input={data ?? null} traceID="">
-      <FlamegraphStateProvider
-        initialState={{
-          preferences: {
-            sorting: 'alphabetical',
-            view: 'bottom up',
-          },
-        }}
-      >
-        <FlamegraphThemeProvider>
-          <Panel>
-            <Flex h={400} column justify="center">
-              {isLoading ? (
-                <LoadingIndicator>{t('Loading Flamegraph')}</LoadingIndicator>
-              ) : isEmpty ? (
-                <EmptyStateWarning>
-                  <p>{t(`A flamegraph isn't available for your query`)}</p>
-                </EmptyStateWarning>
-              ) : (
-                <AggregateFlamegraph />
-              )}
-            </Flex>
-          </Panel>
-        </FlamegraphThemeProvider>
-      </FlamegraphStateProvider>
-    </ProfileGroupProvider>
+    <Flex column gap={space(1)}>
+      <Flex align="center" gap={space(0.5)}>
+        <HeaderTitle>{t('Flamegraph')}</HeaderTitle>
+        <QuestionTooltip
+          size="sm"
+          position="right"
+          isHoverable
+          title={
+            <TooltipContent>
+              <p>{t('An aggregate of profiles for this transaction.')}</p>
+              <p>
+                {t(
+                  'Navigate the flamegraph by scrolling and by double clicking a frame to zoom.'
+                )}
+              </p>
+            </TooltipContent>
+          }
+        />
+      </Flex>
+      <ProfileGroupProvider type="flamegraph" input={data ?? null} traceID="">
+        <FlamegraphStateProvider
+          initialState={{
+            preferences: {
+              sorting: 'alphabetical',
+              view: 'bottom up',
+            },
+          }}
+        >
+          <FlamegraphThemeProvider>
+            <Panel>
+              <Flex h={400} column justify="center">
+                {isLoading ? (
+                  <LoadingIndicator>{t('Loading Flamegraph')}</LoadingIndicator>
+                ) : isEmpty ? (
+                  <EmptyStateWarning>
+                    <p>{t(`A flamegraph isn't available for your query`)}</p>
```

Comment:
Do we know why it wouldn't be available? Like if they select a period where we didn't get any profiles during that time?

Reply:
I see you're reviewing the whitespaced diff 😆 

Yeah basically their page filters didn't yield a result back from vroom. If you set it to something like `1hr`, depending on how frequent the transaction occurs you'll likely get nothing.